acs-core/src/main/java/com/arrow/acs/client/model/ContactModel.java

```diff
@@ -24,6 +24,7 @@
 	private String cell;
 	private String fax;
 	private String monitorExt;
+	private String  middleName; 
```

Comment:
In general, it is desired to re-generate serialVersionUID if change class fields